govcd/system.go

```diff
@@ -108,3 +109,39 @@ func getOrgHREF(vcdClient *VCDClient, orgName string) (string, error) {
 	}
 	return "", fmt.Errorf("couldn't find org with name: %s. Please check Org name as it is case sensitive", orgName)
 }
+
+// Find a list of Virtual Centers matching the filter parameter.
```

Comment:
Please add a list of parameters that can be used and their format

Reply:
Elaborated with additional information